packages/system-tests/assets/sfdx-simple/.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extensions.autoUpdate": false
-}
+    "extensions.autoUpdate": false,
+    "eslint.nodePath": "/home/jsuplizio/DevToolsGit/salesforcedx-vscode/packages/salesforcedx-vscode-lwc/node_modules",
```

Comment:
Sneaky file I'm guessing from running tests, remember to revert.

Reply:
Interesting as I've no idea what made this update or why there's no test failures. I'll make sure this is reverted when I update the PR.

Reply:
This file is reverted with the latest PR. I wasn't aware that running tests locally modified this file.